docs/reference/rest-api/usage.asciidoc

```diff
@@ -360,7 +360,6 @@ GET /_xpack/usage
   }
 }
 ------------------------------------------------------------
-// TESTRESPONSE[skip:Awaits fix https://github.com/elastic/elasticsearch/issues/68101]
```

Comment:
This is still failing.

Reply:
I ran the following test locally, and it succeeded:
```
./gradlew ':docs:integTest' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test"
```